src/networking.c

```diff
@@ -1706,7 +1717,8 @@ void freeClient(client *c) {
 
     /* Log link disconnection with slave */
     if (clientTypeIsSlave(c)) {
-        serverLog(LL_NOTICE,"Connection with replica %s lost.",
+        const char *type = c->flags & CLIENT_REPL_RDB_CHANNEL ? "(rdbchannel)" : "";
+        serverLog(LL_NOTICE,"Connection with replica %s %s lost.", type,
```

Comment:
```suggestion
        const char *type = c->flags & CLIENT_REPL_RDB_CHANNEL ? " (rdbchannel)" : "";
        serverLog(LL_NOTICE,"Connection with replica%s %s lost.", type,
```
maybe we can avoid an extra blank space if not rdb channel